docs/controls/file-browser/v8-schema.md

```diff
@@ -0,0 +1,437 @@
+        };
+        /**
+         * Providing this object indicates that the host app can provide OAuth tokens
+         * via the existing messaging support.
+         */
+        authentication: {};
+        breadcrumb?: {
+            /**
+             * Indicate whether set the root crumb folder path to the entry folder specified in IEntryConfiguration
+             * TODO: deprecated. Should use root to specific the actual root crumb folder path.
+             */
+            setRootToEntry?: boolean;
+
+            /**
+             * Specify the root crumb folder path
+             */
+            root?: {
+                sharePoint?: {
+                    byPath?: {
+                        web?: string;
+                        list?: string;
+                        folder?: string;
+                    };
+                };
+            };
+        };
+        commands?: {
+            /**
+             * Overrides the "open" behavior for files and folders.
+             */
+            open?: {
+                /**
+                 * Specifies a series of 'handlers' for clicking files and folders.
+                 * For each handler, the invoked item is tested against `filters`,
+                 * and the first matching handler has its behavior applied.
+                 * If no handler matches, the default behavior applies.
+                 */
+                handlers?: {
+                    filters?: `.${string}`[];
+                    /**
+                     * Specifies the target for opening the item
+                     * - `none`: Do not allow the item to be opened.
+                     * - `navigate`: Open the item within the experience.
+                     * - `external`: Open the item in a new tab.
+                     * - `host`: Ask the host to open the item.
+                     */
+                    target?: 'none' | 'navigate' | 'external' | 'host';
+                }[];
+            };
+            /**
+             * Overrides behaviors for the view-switch dropdown.
+             */
+            switchView?: {
+                enabled?: boolean;
+                views: {
+                    smartTemplate?: boolean; // default false
+                };
+            };
+            /**
+             * Overrides behaviors for editing SharePoint views.
+             */
+            editView?: {
+                enabled?: boolean;
+            };
+            /**
+             * Overrides behaviors for creating SharePoint views.
+             */
+            createView?: {
+                enabled?: boolean;
+            };
+            /**
+             * Specifies the behavior for closing the experience.
+             */
+            close?: {
+                /**
+                 * A custom label to apply to the 'cancel' button.
+                 * This must be localized by the host app if supplied.
+                 */
+                label?: string;
+            };
+            /**
```

Comment:
Remove `browseThisDevice`, `fromALink`, `switchAccount`, and `manageAccounts`. These are not supported by the File Browser.